tests/extensions/default/analyzers/Microsoft.DotNet.UpgradeAssistant.Extensions.Default.Analyzers.Test/assets/TestClasses/UA0013.Fixed.cs

```diff
@@ -1,10 +1,11 @@
 ﻿using System.Web.Http;
+using System.Web.Mvc;
 using System.Collections.Generic;
 using Microsoft.AspNetCore.Mvc;
 
 namespace TestProject.TestClasses
 {
-    public partial class ValuesController : Controller
+    public partial class ValuesController : ControllerBase
```

Comment:
Why does the fixed version of ApiController now appear to reference System.Web.Mvc.ControllerBase?

Reply:
I changed the code fixer for ApiController to use ControllerBase instead of Controller because I think that's the recommended base type for API controllers in ASP.NET Core.